packages/plugin-fastlane/src/index.ts

```diff
@@ -78,6 +78,18 @@ eval_gemfile(plugins_path) if File.exist?(plugins_path)`,
       },
     );
 
+    // Render Pluginfile template for iOS
+    await withUTF8(
+      path.project.resolve('ios', 'fastlane', 'Pluginfile'),
```

Comment:
Yeah this was a leftover from when I initially had the gems conditional. I decided to revert back to including them both unconditionally in case a project wanted to increment their firebase build number off the appcenter version.